bin/pos-cli.js

```diff
@@ -19,6 +19,7 @@ program
   .command('audit', 'check your code for deprecations, recommendations, errors')
   .command('env', 'manage environments')
   .command('data', 'export, import or clean data on instance')
+  .command('app', 'export the /app directory from the instance')
```

Comment:
Is it:
> export the /app directory from the instance
or 
> export app data to a zip file
?
Because those are two different things in my mind

Reply:
how about `pos-cli pull` ?

Reply:
We already have a `pos-cli pull` command prepared, but it's not available from `pos-cli`. Should I replace it with this one?